01_fib/n0npax/main.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"math"
+	"os"
+)
+
+// Allow to overwtite stream by tests
+var out io.Writer = os.Stdout
+
+// This function is useless and it's provided just due to requirements
```

Comment:
FYI - don't change anything here but the comment `This function is useless` strikes me as strange.
The use of the function `main` in the package `main` is to be automatically run in an executable. It is the entry point of the executable.
When you use `go run main.go` it will print what has been asked to do in this lab.

Reply:
Hi @juliaogris,
Thanks for the review.
My comment wasn't clear. Sorry about that.
I totally understand the purpose of the `main` function. I'm just thinking the `library` may be a better fit for this use-case. The `main` function doesn't consume any input - it's just hardcoded number `7`. In this case, the `main` invokes function `fib`. The `main` is required by lab description, so comment should be rather like: ```//This function is provided due to lab requirements```